scripts/cli_create_kb.py

```diff
@@ -19,7 +19,7 @@ def main(vectors_model: str, language: str):
     logger.info("Constructing knowledge base.")
 
     output_dir = Path(os.path.abspath(__file__)).parent.parent / "output"
-    nlp = spacy.load(vectors_model, exclude=["tagger", "lemmatizer", "attribute_ruler"])
+    nlp = spacy.load(vectors_model, enable=["tok2vec"], disable=[])
```

Comment:
Note that this will still load the other components, in contrast to `exclude`. So with `enable`, the disabled components will still be serialized. Alternatively, we could `exclude` the pipe names that are not `tok2vec`.

Reply:
Ah, but we don't serialize `nlp` anymore. So it's likely not a huge deal (maybe a very slight increase in loading time but probably not worth thinking about)

Reply:
Yeah, exactly. I don't think pipeline loading times are relevant in this context.